src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -16,4 +16,5 @@ public CommandResult execute() {
         addressBook.clear();
         return new CommandResult(MESSAGE_SUCCESS);
     }
+
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.